database/migrations/2020_08_15_172306_create_stand_assignments_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class CreateStandAssignmentsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::statement(
```

Comment:
I'm assuming there is a reason why you can't use the default migrations?

Reply:
Yes - this is something that Laravel have acknowledged but as of yet have announced no intention of fixing, so the workaround is to create the tables using raw SQL. https://github.com/laravel/framework/issues/33238#issuecomment-645965223

Reply:
Added!